elasticsearch/utils/set_sg_replicas

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+source "logging"
+
+info "Modifying the Searchguard index replica settings."
+
+config=${ES_CONF}/elasticsearch.yml
+index=$(python -c "import yaml; print yaml.load(open('${config}'))['searchguard']['config_index_name']" | xargs -i sh -c "echo {}")
+
+info "Disabling auto replication"
+${ES_HOME}/plugins/openshift-elasticsearch/sgadmin.sh \
+    -cd ${HOME}/sgconfig \
+    -i $index \
+    -ks /etc/elasticsearch/secret/searchguard.key \
+    -kst JKS \
+    -kspass kspass \
+    -ts /etc/elasticsearch/secret/searchguard.truststore \
+    -tst JKS \
+    -tspass tspass \
+    -nhnv \
+    -icl \
+    -dra ${DEBUG:+-dg}
+
+info "Updating replica count to 0"
+${ES_HOME}/plugins/openshift-elasticsearch/sgadmin.sh \
+    -cd ${HOME}/sgconfig \
+    -i $index \
+    -ks /etc/elasticsearch/secret/searchguard.key \
+    -kst JKS \
+    -kspass kspass \
+    -ts /etc/elasticsearch/secret/searchguard.truststore \
+    -tst JKS \
+    -tspass tspass \
+    -nhnv \
+    -icl \
+    -us 0 ${DEBUG:+-dg}
+
+info "Setting filters to allocate shard to this node only"
+es_util --query="$index/_settings" -XPUT -d "{\"index.routing.allocation.include._name\": \"$index\"}"
```

Comment:
Before we remove the replicas, shouldn't we first route the primary shard to the same node and then fix up the replicas?